src/StreamService.Models.Dto/StreamService.Models.Dto.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />

```

Comment:
same comment as in StreamService.Broker.csproj

Reply:
it was generated by vs and later fixed